manifests/proxydhcp.pp

```diff
@@ -53,15 +60,24 @@
 
   if $foreman_proxy::dhcp_manage_acls {
 
+    if $foreman_proxy::dhcp_leases =~ /(.*)\/(.*\.leases)/ {
+      $acl_managed_dirs = [$dhcp::dhcp_dir, $1]
+    } else {
+      $acl_managed_dirs = [$dhcp::dhcp_dir]
+    }
+
     package {'acl':
       ensure => 'installed',
     }
+    package {'grep':
+      ensure => 'installed',
+    }
```

Comment:
You can also write this as:

```suggestion
    package { ['acl', 'grep']:
      ensure => 'installed',
    }
```

However, it's probably cleaner to use [ensure_packages](https://forge.puppet.com/puppetlabs/stdlib/reference#ensure_packages):
```suggestion
    ensure_packages(['acl', 'grep'])
```

That has a smaller chance that multiple resources are trying to manage it.